jetpack-beta-admin.php

```diff
@@ -302,6 +302,49 @@ static function show_branch( $header, $branch_key, $branch = null, $section = nu
 		<?php
 	}
 
+	static function show_tag( $header, $tag, $url = null, $section = null, $is_last = false ) {
+		$is_compact = $is_last ? '' : 'is-compact';
+		if ( isset( $url ) ) {
+			$data_tag = sprintf( 'data-tag="%s"', $tag );
+		}
+
+		$className = 'tag-card';
+		list( $current_branch, $current_section ) = Jetpack_Beta::get_branch_and_section();
+		if ( $current_branch === $tag && $current_section === $section ) {
+			$action    = __( 'Active', 'jetpack-beta' );
+			$className = 'tag-card-active';
+		} else {
+			$action = self::activate_button( $tag, $section );
+		}
+
+		$header = str_replace( '-', ' ', $header );
+		$header = str_replace( '_', ' / ', $header );
+		?>
+		<div <?php echo $data_tag; ?> " class="dops-foldable-card <?php echo esc_attr( $className ); ?> has-expanded-summary dops-card <?php echo $is_compact; ?>">
+			<div class="dops-foldable-card__header has-border">
+				<span class="dops-foldable-card__main">
+					<div class="dops-foldable-card__header-text">
+						<div class="dops-foldable-card__header-text tag-card-header">Jetpack <?php echo esc_html( $header ); ?></div>
+						<div class="dops-foldable-card__subheader">
+						<?php
+						sprintf(
+							__( 'Official tag <a href="https://plugins.trac.wordpress.org/browser/jetpack/tags/%s" target="_blank">%s</a> downloaded from wp.org', 'jetpack-beta' ),
```

Comment:
How about something like this instead?

```suggestion
							__( 'Public beta (%2$s) <a href="https://plugins.trac.wordpress.org/browser/jetpack/tags/%1$s" target="_blank" rel="">available on WordPress.org</a>', 'jetpack-beta' ),
```